test/services/ApprovalServiceSpec.scala

```diff
@@ -192,7 +192,7 @@ class ApprovalServiceSpec extends BaseSpec with MockFactory {
     "the processCode exists for another process in the publishedCollection" should {
       "return a DuplicateKeyException" in new Test {
 
-        val expected: RequestOutcome[String] = Right(validId)
+        //val expected: RequestOutcome[String] = Right(validId)
```

Comment:
Can we remove this line if no longer required

Reply:
removed